src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -200,7 +200,7 @@ export function PriceChart({ width, height, tokenAddress, priceData }: PriceChar
         return [
           monthYearFormatter(locale),
           monthYearDayFormatter(locale),
-          timeMonth.range(startDate, endDate, 3).map((x) => x.valueOf() / 1000),
+          timeMonth.range(startDate, endDate, 6).map((x) => x.valueOf() / 1000),
```

Comment:
seems fine, though i'm wondering if it's possible to make this dynamic based on width?

Reply:
Something like that is coming in the next PR for more even spacing of the ticks. This one seemed more urgent though since it's busted and unreadable at the 3mo interval.